src/integration/java/io/pinecone/helpers/BuildUpsertRequest.java

```diff
@@ -39,15 +39,15 @@ public static ArrayList<Float> generateVectorValuesByDimension(int dimension) {
     }
 
     public static ArrayList<Long> generateSparseIndicesByDimension(int dimension) {
-        ArrayList<Long> values = new ArrayList<>();
         Random random = new Random();
+        ArrayList<Long> indices = new ArrayList<>();
```

Comment:
Renamed the `values` field in here, was a little confusing given this function generates indices.